.circleci/config.yml

```diff
@@ -31,6 +26,19 @@ jobs:
             wget -O torchbenchmark.tar.bz2 "https://drive.google.com/u/0/uc?id=1KvYsqipsvvv3pnNkJzME0iTemDZe0buC&export=download&confirm=t"
```

Comment:
@jansel , is it ok to include `1KvYsqipsvvv3pnNkJzME0iTemDZe0buC` as part of the cache key? Or is it a better way to checksum this `torchbenchmark.tar.bz2`?

Reply:
where does this id come from?

Reply:
You can cache it.  If it changes we can update the caching code.